src/shivangi/A13_indexOf.java

```diff
@@ -0,0 +1,64 @@
+package shivangi;
+
+public class A13_indexOf {
+	void AllDuplicateChar(String name){
```

Comment:
method name should starts with small letter

Reply:
Oh sorry @KrishnaKTechnocredits mam it happened by mistake, other than are okay mam?